.github/workflows/ghci.yml

```diff
@@ -0,0 +1,50 @@
+name: GitHub CI tests
+
+on:
+  push:
+    branches: [master, ci, coverity_scan]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos-10.15]
+        cc: [gcc, clang]
+        exclude:
+          - os: macos-10.15
+            cc: gcc
+        include:
+          # Enable distcheck for one build
+          - os: ubuntu-18.04
+            cc: gcc
+            do_distc: yes
+
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+
+      - name: install deps
+        run: |
+          if (echo "${{ matrix.os }}" | grep -qE '^macos-') ; then
+            #brew update
+            # freetype, autoconf and libtool are preinstalled
+            # and `brew install` fails if a non-uptodate version is already installed
```

Comment:
Do you know if Fontconfig is also preinstalled by any chance? It is in Travis CI (but we can neither explicitly `brew install fontconfig` as it complains about the version being out of date nor `brew upgrade fontconfig` as it’s slow).

Reply:
Fontconfig is also [preinstalled](https://github.com/TheOneric/libass/runs/1623767314?check_suite_focus=true#step:4:16).